src/components/hierarchy-bar/hierarchy-bar-node/hierarchy-bar-node.component.html

```diff
@@ -28,14 +38,55 @@
             aria-label="Show children"
             role="button"
             class="hierarchy-bar-node-arrow"
+            [disabled]="readonly"
             [uxPopover]="popoverTemplate"
             [popoverContext]="{ node: node, popover: popover }"
             placement="bottom"
             popoverClass="hierarchy-bar-popover"
             [showTriggers]="hierarchyBar.popoverShowTriggers"
             [hideTriggers]="hierarchyBar.popoverHideTriggers"
             tabindex="0">
-        <ux-icon name="next" class="hierarchy-bar-node-arrow-icon"></ux-icon>
+        <ux-icon name="next"
+                 class="hierarchy-bar-node-arrow-icon">
+        </ux-icon>
+    </button>
+</ng-template>
+
+<ng-template #dropdownMode>
+    <button type="button"
+            uxFocusIndicator
+            uxFocusIndicatorOrigin
+            #popover="ux-popover"
+            aria-label="Show children"
+            class="hierarchy-bar-node-content"
+            [disabled]="readonly"
+            [uxPopover]="popoverTemplate"
+            [popoverContext]="{ node: node, popover: popover }"
+            placement="bottom"
+            popoverClass="hierarchy-bar-popover"
+            [showTriggers]="node.children ? hierarchyBar.popoverShowTriggers : null"
+            [hideTriggers]="node.children ? hierarchyBar.popoverHideTriggers : null"
```

Comment:
As Ashley recommended earlier, use `[]` instead of `null` here.

Reply:
changed